locksmith/config/config.js

```diff
@@ -48,9 +48,4 @@ if (process.env.DATABASE_URL) {
   }
 }
 
-if (process.env.UNLOCK_ENV === 'prod' || process.env.UNLOCK_ENV === 'staging') {
```

Comment:
Hum, are you sure this is not needed?

Reply:
Not required as this is not used by locksmith code anymore. Only sequelize-cli and sequelize migration files. I will add it back anyway so we keep it same.